src/main/webapp/app/entities/event/event-update.tsx

```diff
@@ -34,7 +34,14 @@ export class EventUpdate extends React.Component<IEventUpdateProps, IEventUpdate
 
   componentWillUpdate(nextProps: IEventUpdateProps, nextState: IEventUpdateState) {
     if (nextProps.updateSuccess !== this.props.updateSuccess && nextProps.updateSuccess) {
-      this.handleClose();
+      window.console.log('State after Created: ');
+      window.console.log(nextState.isNew);
+      window.console.log(this.state.isNew);
```

Comment:
still got console log

Reply:
resolved in 03500c3e76568170490cf4692c2fa6453777a438